CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ endif()
 message(STATUS "Shaderc: build type is \"${CMAKE_BUILD_TYPE}\".")
 
 option(SHADERC_ENABLE_SPVC "Enable libshaderc_spvc" OFF)
+option(SHADERC_ENABLE_SPVC_PARSER "Enable spvc parser \(WIP)" OFF)
```

Comment:
nit: I am surprised that ( needs be escaped, but not ), is that correct?

Reply:
@dj2 could you write your explanation here?